### ISSUE TITLE
Error when visiting Review Submissions tab

### ISSUE THREAD

**Issue by luisgreg99** [osulp/Scholars-Archive#765]
Solr Error when visiting `/admin/workflows`
```
ActionView::Template::Error (RSolr::Error::Http - 400 Bad Request
Error: 'no field name specified in query and no default specified via \'df\' param','code'=>400}}
```

The error appears to happen when there is a work with the `pending review` state and a workflow role `approving` is assigned to the user currently logged into the site.

**Comment by luisgreg99** [comment]
It looks like the query being built at https://github.com/samvera/hyrax/blob/master/app/services/hyrax/workflow/status_list_service.rb#L41 needs a default parameter `df` in the query in order to retrieve a valid results from solr.

**Comment by luisgreg99** [comment]
detailed ticket submitted to hyrax https://github.com/samvera/hyrax/issues/1395

**Comment by straleyb** [comment]
We have updated SA to filter the results of the valid(ish) query based on this PR. https://github.com/osulp/Scholars-Archive/pull/851

**Comment by vantuyls** [comment]
@luisgreg99 is this related to https://github.com/osulp/Scholars-Archive/issues/1028?

**Comment by luisgreg99** [comment]
@vantuyls It's unrelated, but it happens in the same section. 

I've added a few notes on what has been changed in the codebase for these tickets here https://github.com/osulp/Scholars-Archive/blob/master/config/initializers/hyrax_status_list_service_hack.rb#L14 

This PR https://github.com/osulp/Scholars-Archive/pull/1035 fixes #1028 

I think we can close this ticket #765 since we have fixed it for SA. It looks like it's still an issue that is being worked on in hyrax samvera/hyrax#1395
